spring-xd-analytics/src/test/java/org/springframework/xd/analytics/metrics/common/RedisRepositoriesConfig.java

```diff
@@ -84,6 +90,18 @@ public StringRedisTemplate stringRedisTemplate() {
 	}
 
 	@Bean
+	public RetryOperations retryOperations() {
+		RetryTemplate retryTemplate = new RetryTemplate();
+		retryTemplate.setRetryPolicy(new SimpleRetryPolicy(3, Collections.<Class<? extends Throwable>, Boolean> singletonMap(RedisConnectionFailureException.class, true)));
+		ExponentialBackOffPolicy backOffPolicy = new ExponentialBackOffPolicy();
+		backOffPolicy.setInitialInterval(1000L);
+		backOffPolicy.setMaxInterval(1000L);
```

Comment:
Can change it if you want, I don't think the tests that use this configuration exercise the retry